components/pkg-export-helm/src/main.rs

```diff
@@ -54,7 +54,7 @@ fn main() {
 }
 
 fn export_for_cli_matches(ui: &mut UI, matches: &clap::ArgMatches) -> Result<()> {
-    let mut chart = Chart::new_for_cli_matches(ui, matches)?;
+    let chart = Chart::new_for_cli_matches(ui, matches)?;
```

Comment:
because of `chart.genarate()` not taking ownership and hence needing mutability IIRC. I only removed mut cause rustc informed me it's not needed anymore.